src/ViewModel/SongLyricViewModel.cpp

```diff
@@ -43,3 +294,37 @@ void SongLyricViewModel::setHasLyric(bool newHasLyric) {
     hasLyric = newHasLyric;
     emit hasLyricChanged();
 }
+
+int SongLyricViewModel::getCurrentplayindex() const {
+    return currentplayindex;
+}
+
+void SongLyricViewModel::setCurrentplayindex(int newcurrentplayindex) {
```

Comment:
```suggestion
void SongLyricViewModel::setCurrentPlayIndex(int newCurrentPlayIndex) {
```

BTW, maybe just `newPlayIndex` instead of `newCurrentPlayIndex`?